fredericoo/saltplay
# feat: admin dashboard

# Overview

A secure, server-side rendered dashboard for all things ~merchant~ player.

## What we have done

- Reorganised API handlers as they had grown in size.
- Created a Higher Order Function to handle authorisation (e.g.: if the user can view the secure pages or not)
- Created pages for editing and adding new offices, games and players, and their respective fields
- Changed some of the pages to be server side rendered (added visual cue loading bar on top)
- Added support for role 'Removed' which replaces the user's name and avatar to Anonymous, whilst still keeping them in the database as the originals

## Blockers/Issues

- Toast is a little wonky on dark mode now, but nothing stopping us from progressing.

## How to test

- Be me
- Be an admin
- Go on dashboard
- Add random games and offices for the lulz
- See games appear
- ?????????
- profit

## Checklist:
